wavefront_cli/commands/configure.py

```diff
@@ -11,5 +11,5 @@ class Configure(Base):
 
     def run(self):
 
-        print "The configure command will overwrite the Wavefront URL and API Token stored in ~/.wavefront/credentials"
-        auth.do_auth(self.options)
\ No newline at end of file
+        print("The configure command will overwrite the Wavefront URL and API Token stored in ~/.wavefront/credentials")
```

Comment:
Did we consider using `logging.info` instead of `print()` everywhere in this tool?

Reply:
I am not sure about using logger, as there are very few places where a logger can be used as this runs as an interactive terminal, and prints the messages in the terminal. There is a dedicated class  **wavefront_cli/lib/message.py** for printing the messages.